Content.Server/Psionics/Telepathy/TelepathicRepeaterComponent.cs

```diff
@@ -1,4 +1,4 @@
-namespace Content.Server.Nyanotrasen.Chat
+namespace Content.Server.Psionics.Telepathy
```

Comment:
i thought this wasn't ported to Delta?

This should be used to re-port the "Psionic Orb"

Reply:
This is a component that gets temporarily added when anyone consumes hallucinogens. It's what makes you repeat telepathy.